### DISCUSSION TITLE
Will @clerk/elements support other frameworks than Next.js?

### DISCUSSION THREAD

**Discussion by depsimon** [clerk/javascript#2856] [Support request - DO NOT OPEN HERE] [answered]
It seems that at the moment it only supports Next.js. Is there any plans to support other frameworks or even make it framework-agnostic?

I'm particularly interested in using it in Remix to build my own auth pages.

**Comment by LekoArts** [comment, marked as answer]
To bring everyone to the same page, this is what Clerk Elements is about: https://feedback.clerk.com/roadmap/b79da4a2-694c-4468-bea2-fb1d3570601a
You can upvote the feature request to get notified about updates. Or check out our https://clerk.com/changelog for news.

To answer your question: Yes, it'll support other React frameworks short-medium-term! Framework-agnostic is a long-term goal but that's far into the future.
